### PR TITLE
Replaced from 'g8' to 'sbt new'

### PR DESCRIPTION
**Pull request by magnolia-k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ $ sbt
 > exit
 $ cd target/sbt-test/default-*/scripted
 $ sbt
-$ jetty:start
-$ browse # starts browser for you, or manually open http://localhost:8080 to verify
+> jetty:start
+-> executes browser, and opens `http://localhost:8080` to verify
```
Comment (takezoe) [inline README.md]: This line means to run `browse` task which is defined in the Scalatra sbt plugin. This task opens the application URL by the default web browser.
https://github.com/scalatra/scalatra-sbt/blob/03338c12b46fc6a609ffdb9fcb25cc24cf9db230/src/main/scala/org/scalatra/sbt/ScalatraPlugin.scala#L14

I think this description should be kept.

Comment (magnolia-k) [inline README.md]: When trying according to the procedure, it seems that at the time of jetty:start, the sbt command line is occupied and there is no timing to start the 'browse' command.

Another sbt is launched and the browse command displays the following error.
```
Not a valid command: browse (similar: set)
```

Comment (takezoe) [inline README.md]: You can hit `browse` after `jetty:start` -> `Enter`. This is my result after `g8Test`:
```bash
$ cd target/sbt-test/scalatra-sbt-g8/scripted/
$ sbt
...
> jetty:start
...
[success] Total time: 8 s, completed 2017/06/05 20:48:04
...
> browse
[info] Launching browser.
[success] Total time: 0 s, completed 2017/06/05 20:48:16
```
Then my browser was launched.

Comment (magnolia-k) [inline README.md]: I'm sorry.
My operation was wrong ...
However, since the path names are different, we will fix that.

**Comment by takezoe** [comment]
@magnolia-k Thanks for quick fix!

**Pull request merged**